gui/GameConqueror.py

```diff
@@ -809,8 +809,9 @@ def select_process(self, pid, process_name):
             self.process_label.set_property('tooltip-text', _('Select a process'))
             self.show_error(_('Cannot retrieve memory maps of that process, maybe it has '
                               'exited (crashed), or you don\'t have enough privileges'))
-        self.process_label.set_text('%d - %s' % (pid, process_name))
-        self.process_label.set_property('tooltip-text', process_name)
+        else:
+            self.process_label.set_text('%d - %s' % (pid, process_name))
+            self.process_label.set_property('tooltip-text', process_name)
 
         self.command_lock.acquire()
         self.backend.send_command('pid %d' % (pid,))
```

Comment:
Please write your current state description in the commit description in present. You're writing it for the maintainer and before merging, it is not yet fixed. I've changed the description and merged this. Thanks!